lua/entities/gmod_wire_expression2/core/custom/cl_ftrace.lua

```diff
@@ -1,137 +1,164 @@
+E2Helper.Descriptions["setFilterEar(xft:)"] = "Changes the filtering mode to entity array"
+E2Helper.Descriptions["setFilterEnu(xft:)"] = "Changes the filtering mode to entity unit"
+E2Helper.Descriptions["setFilterFnc(xft:)"] = "Changes the filtering mode to function routine"
+E2Helper.Descriptions["setIsIgnoreWorld(xft:n)"] = "Updates the flash tracer trace `IgnoreWorld` flag"
+E2Helper.Descriptions["setLen(xft:n)"] = "Updates flash tracer length distance"
+E2Helper.Descriptions["setMask(xft:n)"] = "Updates flash tracer trace hit mask enums MASK"
+E2Helper.Descriptions["setPos(xft:nnn)"] = "Updates the flash tracer origin position using three numbers"
+E2Helper.Descriptions["setPos(xft:r)"] = "Updates the flash tracer origin position using an array"
+E2Helper.Descriptions["setPos(xft:v)"] = "Updates the flash tracer origin position using a vector"
+E2Helper.Descriptions["smpLocal(xft:)"] = "Samples the flash tracer and updates the trace result by base attachment entity local axis"
+E2Helper.Descriptions["smpLocal(xft:a)"] = "Samples the flash tracer and updates the trace result by base position, angle"
+E2Helper.Descriptions["smpLocal(xft:e)"] = "Samples the flash tracer and updates the trace result by entity position and forward vectors"
+E2Helper.Descriptions["smpLocal(xft:ea)"] = "Samples the flash tracer and updates the trace result by entity position, angle"
+E2Helper.Descriptions["smpLocal(xft:ev)"] = "Samples the flash tracer and updates the trace result by position, entity angle"
+E2Helper.Descriptions["smpLocal(xft:v)"] = "Samples the flash tracer and updates the trace result by position, base angle"
+E2Helper.Descriptions["smpLocal(xft:va)"] = "Samples the flash tracer and updates the trace result by position, angle"
+E2Helper.Descriptions["smpWorld(xft:)"] = "Samples the flash tracer and updates the trace result by the world axis"
+E2Helper.Descriptions["smpWorld(xft:a)"] = "Samples the flash tracer and updates the trace result by entity position and angle forward"
+E2Helper.Descriptions["smpWorld(xft:e)"] = "Samples the flash tracer and updates the trace result by entity position and forward vectors"
+E2Helper.Descriptions["smpWorld(xft:ea)"] = "Samples the flash tracer and updates the trace result by entity position, angle"
+E2Helper.Descriptions["smpWorld(xft:ev)"] = "Samples the flash tracer and updates the trace result by position, entity angle"
+E2Helper.Descriptions["smpWorld(xft:v)"] = "Samples the flash tracer and updates the trace result by position vector and entity forward"
+E2Helper.Descriptions["smpWorld(xft:va)"] = "Samples the flash tracer and updates the trace result by position, angle"
+E2Helper.Descriptions["updEarSZ(xft:)"] = "Performs flash tracer entity array filter list refresh"
+E2Helper.Descriptions["useFilterEar(xft:xft)"] = "Uses other flash tracer entity array filtering by reference"

```

Comment:
Ear is also a terrible name. Just replace Ear with Array. People do not like remembering what abbreviations stand for.

Reply:
And others to be Function/Entity since I am already expanding ?